app/models/profile.rb

```diff
@@ -37,27 +37,40 @@ def gpo_verification_pending?
     gpo_verification_pending_at.present?
   end
 
+  def pending_reasons
+    [
+      *(:gpo_verification_pending if gpo_verification_pending?),
+      *(:fraud_check_pending if has_fraud_deactivation_reason?),
```

Comment:
I changed this to `has_fraud_deactivation_reason?` because it also checks if threatmetrix is enabled.